contracts/PriceFeed/PriceFeed.sol

```diff
@@ -181,6 +246,77 @@ contract PriceFeed is Initializable, ProxyOwned {
         if (!hasCurrencyKey) {
             currencyKeys.push(currencyKey);
         }
+        _updateStaticPricePerAsset(currencyKey, rate);
+    }
+
+    /**
+     * @notice Update the static price for a given asset.
+     * @dev The caller must be whitelisted or the owner. Updates are allowed only if at least one day has passed since the last update and the new rate does not exceed the permitted increase.
+     * @param currencyKey The identifier of the asset.
+     * @param rate The new static price.
+     */
+    function updateStaticPricePerAsset(bytes32 currencyKey, uint216 rate) external {
+        require(whitelistedAddresses[msg.sender] || msg.sender == owner, "Only whitelisted can set static price");
+        uint40 currentUpdateTime = staticPricePerAsset[currencyKey].time;
+        uint216 currentRate = staticPricePerAsset[currencyKey].rate;
+        require(uint40(block.timestamp) - currentUpdateTime > rateUpdateInterval, "Rate update too frequent");
+        require(
```

Comment:
the case of currentRate==rate is not covered, perhaps its best to throw a require error in that case